// dabi-images/tags/nsfw/real.js
const requestHandler = require("../handler");
class RealShit {
    butts() {
        let subreddits = ["ass", "butt", "butts", "cosplaybutts", "girlsinyogapants",
                "smalltitsbigass", "booty", "AsianAss", "WhiteCheeks", "HungryButts",
                "ASSians", "brunetteass", "Redheadass", "beautifulbutt", "pawg"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    boobs() {
        let subreddits = ["Boobies", "tits", "ToplessInPanties", "boobs", "PerfectTits", "adorableporn", "amazingtits", "topless",
            "fortyfivefiftyfive", "BustyPetite", "Titties", "NSFWfashion", "Playboy", "PerfectBoobs", "bodyperfection"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    vag() {
        let subreddits = ["cameltoe", "ExtremeCameltoe", "Innie", "vagina", "AsianPussy", "Innies", "GodPussy",
                            "shavedpussies", "PerfectPussies", "simps", "rearpussy"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
        console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    wiener() {
        let subreddits = ["MassiveCock", "blackcock",  "bigdicks", "penis", "BonersInPublic", "CumCocks", "penislover",
        "cock", "softies", "Blackdick", "Boners", "NSFW_DICK_and_Cock", "WhipItOut", "sizecomparison",
        "handsfree", "PENIS_PENIS_PENIS", "CutCocks", "dicks", "dickpic"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    porn() {
        let subreddits = ["gonewild", "nsfw", "NSFW_GIF", "RealGirls", "cumsluts", "nsfw_gifs", "GirlsFinishingTheJob", "porninfifteenseconds",
        "60fpsporn", "canthold", "FunWithFriends", "Blowjobs", "edging", "porn", "whenitgoesin", "Facials"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    buttgifs() {
        let subreddits = ["assgifs", "BootyGIFs", "AssReveal", "LovetoWatchYouLeave", "girlswhoride", "tightsqueeze", "booty_gifs",
                          "twerking"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    boobsgifs() {
        let subreddits = ["boobbounce", "TittyDrop", "CasualJiggles", "SloMoBoobs", "GifsOfRemoval", "BouncingTits", "stripgirls", "BiggerThanYouThought"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    hentai() {
        let subreddits = ["hentai", "waifusgonewild", "GameOverGirls", "MaidHentai", "anime_nsfw", "Naruto_Hentai", "HentaiAnime", 
        "BigAnimeTiddies", "uncensoredhentai", "adorablehentai", "swimsuithentai", "Bleach_Hentai", "NSFW_Hentai_pics", "AnimeBooty", 
        "thick_hentai", "Overwatch_Porn", "rule34"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    funny() {
        let subreddits = ["memes", "dankmemes", "okbuddyretard", "comedyheaven", "gamingmemes",
                          "me_irl", "holdmybeer", "funny", "Funnypics", "therewasanattempt", "AnimalsFailing",
                          "Whatcouldgowrong", "AnimalsBeingJerks", "Unexpected"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    yugioh() {
        let subreddits = ["yugiohcollections", "yugiohshowcase"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    pokemon() {
        let subreddits = ["pkmntcgcollections", "PokemonTCG"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    food() {
        let subreddits = ["FoodPorn", "food", "burgers", "pizza", "eatsandwiches", "tonightsdinner", "southernfood", "steak",
                          "DessertPorn", "sushi", "BBQ", "PutAnEggOnIt", "appetizers", "cajunfood", "mexicanfood", "grilledcheese",
                          "ramen", "JapaneseFood"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    cosplay() {
        let subreddits = ["cosplaygirls", "cosplay", "CosplayBoobs", "gwcosplay", "cosplaybabes", "CosplayLewd", "nsfwcosplay",
                          "geekygirls", "CosplayGals", "cosplaybutts", "CosplayPanties", "belledelphine_", "suicidegirls", "SuicideGirls"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
}
module.exports = new RealShit();
