// dabi-images/tags/nsfw/real.js
const requestHandler = require("../handler");
class RealShit {
    butts() {
        let subreddits = ["ass", "CuteLittleButts", "butt", "butts", "cosplaybutts", "girlsinyogapants",
                "smalltitsbigass", "booty", "beautiful_asses", "AsianAss", "WhiteCheeks",
                "ASSians", "AsianAsses", "CuteLittleButts", "brunetteass", "Redheadass"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    boobs() {
        let subreddits = ["Titties", "topless", "Boobies", "tits", "ToplessInPanties",
            "perkytits", "PerfectWhiteBoobs","NiceTitties", "boobs", "BeautifulTitsAndAss", "Topless_Vixens",
            "C_Cups", "RedditsBestTits", "Perky","PerfectTits", "Perfect_Tits"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    vag() {
        let subreddits = ["cameltoe", "ExtremeCameltoe", "pussy", "Innie",
                            "vagina", "AsianPussy", "Innies", "shavedgirls", "PerfectPussy", "GodPussy",
                            "shavedpussies", "PerfectPussies", "simps"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    wiener() {
        let subreddits = ["MassiveCock", "blackcock",  "bigdicks", "penis", "BonersInPublic", "CumCocks", "penislover",
        "cock", "softies", "Blackdick", "Boners", "NSFW_DICK_and_Cock", "WhipItOut", "sizecomparison",
        "handsfree", "PENIS_PENIS_PENIS", "CutCocks", "dicks", "dickpic"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    porn() {
        let subreddits = ["NSFW_GIF", "nsfw_gifs", "porn_gifs", "porninfifteenseconds", "CuteModeSlutMode", "60fpsporn", 
        "NSFW_HTML5", "the_best_nsfw_gifs", "verticalgifs", "PornGifs", "gifsgonewild", "sexgifs", "porngifsonly"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    buttgifs() {
        let subreddits = ["assgifs", "BootyGIFs", "AssReveal", "LovetoWatchYouLeave", "girlswhoride", "tightsqueeze", "booty_gifs"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]); 
    }
    boobsgifs() {
        let subreddits = ["boobgifs", "boobbounce", "boob_gifs", "feelthemup", "burstingout", "FlashingGirls", "TittyDrop"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    hentai() {
        let subreddits = ["hentai", "waifusgonewild", "GameOverGirls", "MaidHentai", "anime_nsfw", "Naruto_Hentai", "HentaiAnime", 
        "BigAnimeTiddies", "uncensoredhentai", "adorablehentai", "swimsuithentai", "Bleach_Hentai", "NSFW_Hentai_pics", "AnimeBooty", "thick_hentai"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
    meme() {
        let subreddits = ["memes", "wholesomememes", "dankmemes", "historymemes", "lastimages", "okbuddyretard", "comedyheaven", 
                          "MeanJokes", "tooson", "Teleshits", "imgoingtohellforthis", "ImAHorriblePerson", "nottheonion", "bannedfromclubpenguin",
                          "copypasta", "me_irl", "shittylifeprotips"];
        return requestHandler.makeRequest("reddit", subreddits[Math.floor(Math.random() *subreddits.length)]);
    }
}
module.exports = new RealShit();
