// dabi-images/tags/nsfw/real.js
const requestHandler = require("../handler");
class RealShit {
    butts() {
        let subreddits = ["ass", "CuteLittleButts", "butt", "butts", "cosplaybutts", "girlsinyogapants",
                "smalltitsbigass", "booty", "beautiful_asses", "AsianAss", "WhiteCheeks",
                "ASSians", "AsianAsses", "CuteLittleButts", "brunetteass", "Redheadass"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    boobs() {
        let subreddits = ["Titties", "topless", "Boobies", "tits", "ToplessInPanties",
            "perkytits", "PerfectWhiteBoobs","NiceTitties", "boobs", "BeautifulTitsAndAss", "Topless_Vixens",
            "C_Cups", "RedditsBestTits", "Perky","PerfectTits", "Perfect_Tits", "adorableporn"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    vag() {
        let subreddits = ["cameltoe", "ExtremeCameltoe", "Innie",
                            "vagina", "AsianPussy", "Innies", "GodPussy",
                            "shavedpussies", "PerfectPussies", "simps", "rearpussy"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
        console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    wiener() {
        let subreddits = ["MassiveCock", "blackcock",  "bigdicks", "penis", "BonersInPublic", "CumCocks", "penislover",
        "cock", "softies", "Blackdick", "Boners", "NSFW_DICK_and_Cock", "WhipItOut", "sizecomparison",
        "handsfree", "PENIS_PENIS_PENIS", "CutCocks", "dicks", "dickpic"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    porn() {
        let subreddits = ["NSFW_GIF", "nsfw_gifs", "porn_gifs", "porninfifteenseconds", "CuteModeSlutMode", "60fpsporn", 
        "NSFW_HTML5", "the_best_nsfw_gifs", "verticalgifs", "PornGifs", "gifsgonewild", "sexgifs", "porngifsonly"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    buttgifs() {
        let subreddits = ["assgifs", "BootyGIFs", "AssReveal", "LovetoWatchYouLeave", "girlswhoride", "tightsqueeze", "booty_gifs"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    boobsgifs() {
        let subreddits = ["boobgifs", "boobbounce", "boob_gifs", "feelthemup", "burstingout", "FlashingGirls", "TittyDrop"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    hentai() {
        let subreddits = ["hentai", "waifusgonewild", "GameOverGirls", "MaidHentai", "anime_nsfw", "Naruto_Hentai", "HentaiAnime", 
        "BigAnimeTiddies", "uncensoredhentai", "adorablehentai", "swimsuithentai", "Bleach_Hentai", "NSFW_Hentai_pics", "AnimeBooty", "thick_hentai"];
        const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
        return requestHandler.makeRequest("reddit", subreddit);
    }
    meme() {
        let subreddits = ["memes", "wholesomememes", "dankmemes", "historymemes", "okbuddyretard", "comedyheaven", 
                          "MeanJokes", "toosoon", "Teleshits", "imgoingtohellforthis", "me_irl", "ShittyLifeProTips"];
            const subreddit = subreddits[Math.floor(Math.random() *subreddits.length)];
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    yugioh() {
        let subreddit ="yugiohcollections";
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
    pokemon() {
        let subreddit ="pkmntcgcollections";
            console.log(subreddit);
            return requestHandler.makeRequest("reddit", subreddit);
    }
}
module.exports = new RealShit();
